app/scripts/controllers/account-order.ts

```diff
@@ -3,6 +3,7 @@ import {
   RestrictedMessenger,
   StateMetadata,
 } from '@metamask/base-controller';
+import { AccountOrderControllerGetStateAction } from '../controller-init/messengers/account-order-controller-messenger';
```

Comment:
This cause a circular dep, see:
- https://github.com/MetaMask/metamask-extension/pull/37017/files#r2454402074